tests/jenkins/install.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/bash
+
+. /gpfs/bbp.cscs.ch/apps/hpc/jenkins/config/modules.sh
+
+CORENRN_TYPE=$1
+
+if [ ${CORENRN_TYPE} = "GPU" ]
+then
+    module load pgi cuda hpe-mpi cmake
+
+    mkdir build_${CORENRN_TYPE}
+else
+    module load boost intel hpe-mpi cmake
+    export CC=mpicc
```

Comment:
We dont need these exports for the GPU version?

Reply:
You are right. The exports are needed to use the `hpe-mpi` compiler and are included in the tutorial of building `CoreNeuron for GPU`, however if they are not set the `PGI` compiler is used and `CoreNeuron` is still built without problems and without any big difference in performance.

Reply:
CMake will handle both cases for you. (sorry I lost the source of this info...)
If you `export CC=mpicc` it will use the mpi compiler wrapper, if not, it will use the CXX compiler and detect the MPI flags for you.
I believe it would be better to be consistent, but I don't know if there are any gotchas. @pramodk ?